src/js/helpers/aboutTheDataHelper.js

```diff
@@ -18,6 +73,101 @@ export const aboutTheDataQueryString = (params) => {
     ${params.sort ? `&sort=${params.sort}` : ''}`;
 };
 
+const defaultState = {
+    page: 1,
+    limit: 10
+};
+
+export const usePagination = (initialState = defaultState) => {
+    const [{ page, limit }, updatePagination] = useState(initialState);
+    return [{ page, limit }, updatePagination];
+};
+
+export const getTotals = (fy = '2020', period = '12') => {
```

Comment:
we should follow our convention for api calls and be more descriptive `fetchTotalBudgetaryResources`